src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Claims/WindowsClaimSet.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace CoreWCF.IdentityModel.Claims
 {
-    internal class WindowsClaimSet : ClaimSet, IIdentityInfo, IDisposable
+    public class WindowsClaimSet : ClaimSet, IIdentityInfo, IDisposable
     {
```

Comment:
We need to be thoughtful about how we make this class public. There's a lot of overloads of the constructor and we might not want to expose them all. I don't see a reason to make this public right now. New api surface also triggers bumping the minor version up to 1.1 and I want to avoid that for now if possible.

Reply:
Thanks for catching this. This change is an artifact from a previous implementation which I eventually scrapped. Will revert.